script/validators/source_validator.py

```diff
@@ -51,6 +52,9 @@
     os.path.join(PELOTON_DIR, "test")
 ]
 
+# To be used in check_includes.
+PATHS = set([path.lstrip('src/include') for path in glob.glob('src/include/**/*.h')])
```

Comment:
`lstrip` doesn't work as intended here, as it takes a set of characters, and not a string. Use string replace or just cut off the first x characters array access.